Adds the disk storage fridge

SingingSpock opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
<details>
<summary>
	 Adds a disk storage machine
</summary>
<hr>

Adds a subtype of smartfridge that holds disks, and maps them in at the places where a table/rack that held a faction's disks used to be, as well as the public lathe. More can be made with a board found on the Guild Circuits disk. Also adds the ability to make regular smartfridges for good measure, since those couldn't be made previously.
	
![image](https://github.com/sojourn-13/sojourn-station/assets/29682682/2920cf7b-cfda-4109-b99a-a50253902023)

<hr>
</details>

## Changelog
:cl:
add: New disk storage fridges
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

benj8560 commented:
minor stuffs but the table next to the prospectors lathe has been rendered inaccessible.
![dreamseeker_9Kw264TrDY](https://github.com/sojourn-13/sojourn-station/assets/26511091/4352def7-f88a-4e86-9656-7b585564ef5c)

 Theres also a recharger causually lodged into the guilds disk storage.
![dreamseeker_luWaJh0RDx](https://github.com/sojourn-13/sojourn-station/assets/26511091/f48ab90a-3d95-4c5e-9d3f-0463bcbcb975)

benj8560 commented:
on another note. I believe the server was called Oculus a smaller Eris based server that had this exact same idea. They should have sprites that work for these I recall them looking akin floppy disk organizer pretty much exactly what they need to be. Made to be on table.

SingingSpock commented:
Swapped the positions of the disk fridge and the table in prospies, moved the charger over to the table in Guild, and added a disk storage to the Steward office since I missed they get their own lathe. As far as sprites from another server, I have very little experience grabbing/integrating such but someone else could definitely do it after merge
